### PR TITLE
Allow project_root_path and verbosity to come from Django settings

### LINKED ISSUE #203: Allow project_root_path to be read from settings.py
In the latest change in #199 that fixes #198, we can now get all settings from Django's settings, _except_ for `project_root_path` and `verbosity`. This is because of how early it's read in the process:

https://github.com/3YOURMIND/django-migration-linter/blob/8e56562771df63094349fb667bdb06a872462499/django_migration_linter/management/commands/lintmigrations.py#L141-L146

In my case, my project layout looks like this:

```
.
├── Dockerfile
├── manage.py
├── myapp
│   ├── configuration
│   │   ├── __init__.py
│   │   ├── settings.py
```

And in `manage.py` I default the `DJANGO_SETTINGS_MODULE`:

```python
...
if __name__ == "__main__":
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "myapp.configuration.settings")
    try:
        from django.core.management import execute_from_command_line
    ...
```

In this case, there is a loose relationship between my project root path and where `settings.py` is (it's actually 2 level up). I think the `lintmigrations` management command assumes that they are the same.

The manifestation of this is that the migration linter comes back with 0 migrations to check, since it's effectively starting at `myapp/configuration`:

```
*** Summary ***
Valid migrations: 0/0
Erroneous migrations: 0/0
Migrations with warnings: 0/0
Ignored migrations: 0/0
```

### PR DESCRIPTION
**Pull request by phillipuniverse**
Fixes #203

### CODE REVIEW

**Comment by phillipuniverse** [comment]
@David-Wobrock I was looking at the other issues and this might also be a valid solution for #141

File: django_migration_linter/management/commands/lintmigrations.py [hunk 1]
```diff
@@ -157,8 +147,10 @@ def handle(self, *args, **options):
             all_warnings_as_errors,
         ) = extract_warnings_as_errors_option(options["warnings_as_errors"])
 
+        configure_logging(options["verbosity"])
+
         linter = MigrationLinter(
-            settings_path,
+            options["project_root_path"],
```
[the remark this answers is not in the document]
Comment (phillipuniverse) [reply django_migration_linter/management/commands/lintmigrations.py:153]: Ah good catch. Currently this code has backwards-incompatible changes as it requires a user to set project_root_path when before they didn’t.

I’ll make this change!

[the remark this answers is not in the document]
Comment (David-Wobrock) [reply django_migration_linter/management/commands/lintmigrations.py:153]: Awesome, thanks!
You might have noticed that I took the liberty to rebase against the latest `main`

[the remark this answers is not in the document]
Comment (phillipuniverse) [reply django_migration_linter/management/commands/lintmigrations.py]: @David-Wobrock I did notice and thanks! This is now complete

[the remark this answers is not in the document]
Comment (David-Wobrock) [reply django_migration_linter/management/commands/lintmigrations.py]: Black won't be happy about the new line I think :P
Either just format it, or put it in a variable to give it a little name that can make sense :)

**Review by David-Wobrock — APPROVED** [review]

**Comment by David-Wobrock** [comment]
Thanks a lot for the PR @phillipuniverse !

**Pull request merged**